packages/clients/test/aptos/gas.integration.test.ts

```diff
@@ -23,7 +27,7 @@ const gas = createGasClient();
 
 const account = Account.generate({
   scheme: SigningSchemeInput.Ed25519,
-  legacy: false,
+  legacy: true,
```

Comment:
Just curious, why is this now legacy?

Reply:
This was changed because their JSON submission API cannot deal with non-legacy keys at the moment due to buggy JSON serde.